kits/mou-drafter/apps/public/swiftlatex/editor.html

```
@@ -0,0 +1 @@
+We are working hard to fix the editor. 
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)